commune_app/all_models/users.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
```

Comment:
@FaezZangariya when you look at the "upstream repo" in:
https://github.com/redhat-beyond/commune/blob/main/commune_app/all_models/users.py
you can see that this file already merged
Please rebase your "origin repo" to be on the same page as the "upstream repo"